common/consts.go

```diff
@@ -1,11 +1,7 @@
 package common
 
 const (
-	MainnetChainID = 1 //nolint
-	GorliChainID   = 5
-	GanacheChainID = 1337
-
-	DefaultMoneroDaemonMainnetPort  = 18081
```

Comment:
Apparently, the `nolint` was getting applied to all the elements in the `const` block, not just the elements before the blank line.